src/styles/main.scss

```
@@ -12,3 +12,35 @@ body {
 
 @import "_header.scss";
 @import "_catalog.scss";
+
+@media (min-width: 1px) and (max-width: 487px) {
+  .catalog {
+    display: grid;
+    gap: 48px;
+    grid-template-columns: repeat(1, 200px);
+  }
+}
+
+@media (min-width: 488px) and (max-width: 767px) {
```

```suggestion
@media (max-width: 767px) {
```
не обов'язково прописувати скрізь мінімальну ширину. В тебе інші медіа-запити це перекривають